client/graph.c

```diff
@@ -18,16 +18,16 @@
 int GraphBuffer[MAX_GRAPH_TRACE_LEN];
 int GraphTraceLen;
 
-/* write a bit to the graph */
+/* write a manchester bit to the graph */
 void AppendGraph(int redraw, int clock, int bit)
 {
   int i;
-
+  //set first half the clock bit (all 1's or 0's for a 0 or 1 bit) 
```

Comment:
since it is working with the int GraphBuffer[] and not a uint_8 or char, i'm not sure how to make memset work properly...  nor is it worth trying to figure it out ;)